src/ui/server/helpers/has-form-data/index.ts

```diff
@@ -0,0 +1,53 @@
+import { RequestBody } from '../../../types';
+
+/**
+ * stripEmptyFormFields
+ * Filters out form fields with empty values
+ * @param {Express.Request.body} Form data
+ * @returns {Object} Form data without empty values
+ */
+export const stripEmptyFormFields = (formBody: RequestBody) => {
+  if (formBody) {
+    const fieldsWithValues = {};
+
+    const keys = Object.keys(formBody);
+
+    keys.forEach((key) => {
+      const value = formBody[key];
+
+      if (value !== '' && value !== null && value !== undefined) {
```

Comment:
Might be efficient to write `Boolean(value)`

Reply:
Great shout, changed to this and allowing for false boolean as well